@navikt/core/tokens/darkside/tokens/semantic-roles.ts

```diff
@@ -2,127 +2,167 @@ import _ from "lodash";
 import {
   type ColorTheme,
   type GlobalColorRoles,
+  StyleDictionaryToken,
   type StyleDictionaryTokenConfig,
+  TokenTypes,
   globalColorRoles,
 } from "../util";
 
-const configForRole = (
-  role: GlobalColorRoles,
-  theme: ColorTheme,
-): StyleDictionaryTokenConfig<"color"> => ({
+type BgKeysT =
+  | GlobalColorRoles
+  | `${GlobalColorRoles}-hover`
+  | `${GlobalColorRoles}-hoverA`
+  | `${GlobalColorRoles}-moderate`
+  | `${GlobalColorRoles}-moderateA`
+  | `${GlobalColorRoles}-moderate-hover`
+  | `${GlobalColorRoles}-moderate-hoverA`
+  | `${GlobalColorRoles}-moderate-pressed`
+  | `${GlobalColorRoles}-moderate-pressedA`
+  | `${GlobalColorRoles}-strong`
+  | `${GlobalColorRoles}-strong-hover`
+  | `${GlobalColorRoles}-strong-pressed`
+  | `${GlobalColorRoles}-raised`
+  | `${GlobalColorRoles}-raised-hover`;
+
+type TextKeysT =
```

Comment:
Was it intentional to use `T` suffix on these types? I don't have any strong opinions about it, but we usually don't do it.